lib/styleToFilters.js

```diff
@@ -173,4 +193,29 @@ function getPropertyFromExpression(exp, properties) {
   }
 }
 
-module.exports = styleToFilters;
\ No newline at end of file
+// cli tool
+if (require.main === module) {
+  var fs = require('fs');
+  var path = require('path');
+  var JSONStream = require('JSONStream');
+
+  if (process.argv.length === 3) {
+    var styleString = fs.readFileSync(path.resolve(process.argv[2]));
+    var style = JSON.parse(styleString);
+    console.log(JSON.stringify(styleToFilters(style)));
+  } else if (process.argv.length === 2) {
+    process.stdin
+      .pipe(JSONStream.parse())
+      .on('data', function(style) {
+        console.log(JSON.stringify(styleToFilters(style)));
+      });
+  } else {
+    throw new Error('wrong number of arguments\n' +
+      'Usage:\n' +
+      '  node ./lib/styles-to-filter.js ./fixtures/style.json\n' +
+      'or\n' +
+      '  node ./lib/styles-to-filter.js < ./fixtures/style.json');
+  }
+}
```

Comment:
@lukaswelte mind removing this? I think it is covered by https://github.com/mapbox/vtshaver/blob/master/bin/vtshaver-filters.js, right?